ownership-chain/runtime/src/tests/xcm_tests.rs

```diff
@@ -0,0 +1,165 @@
+//! Cross-chain asset transfer tests
+
+use super::xcm_mock::{parachain::Assets, MockNet, ParaA, ParaB, Relay};
+use crate::{
+	tests::xcm_mock::{
+		msg_queue::mock_msg_queue, parachain, ParachainBalances, ParachainXtokens,
+		RelayChainPalletXcm, ALITH, BOBTH, INITIAL_BALANCE,
+	},
+	UNIT,
+};
+use cumulus_primitives_core::{
+	Fungibility::Fungible,
+	Instruction::{BuyExecution, Transact, WithdrawAsset},
+	Junction::{AccountKey20, Parachain},
+	Junctions::{Here, X1, X2},
+	MultiAsset, MultiLocation, NetworkId, OriginKind,
+	WeightLimit::Unlimited,
+	Xcm,
+};
+use frame_support::{assert_ok, traits::fungibles::Inspect, weights::Weight};
+use parity_scale_codec::Encode;
+use xcm_simulator::TestExt;
+
+/// Test downward message passing. Does some basic remark in the parachain from the relay chain.
+#[test]
+fn basic_dmp() {
+	MockNet::reset();
+
+	let remark = parachain::RuntimeCall::System(
+		frame_system::Call::<parachain::Runtime>::remark_with_event { remark: vec![1, 2, 3] },
+	);
+
+	Relay::execute_with(|| {
+		assert_ok!(RelayChainPalletXcm::send_xcm(
+			Here,
+			Parachain(1),
+			Xcm(vec![
+				WithdrawAsset(
+					vec![MultiAsset {
+						id: MultiLocation::here().into(),
+						fun: cumulus_primitives_core::Fungibility::Fungible(UNIT)
+					}]
+					.into()
+				),
+				BuyExecution {
+					fees: MultiAsset {
+						id: MultiLocation::here().into(),
+						fun: cumulus_primitives_core::Fungibility::Fungible(UNIT)
+					},
+					weight_limit: Unlimited,
+				},
+				Transact {
+					origin_kind: OriginKind::SovereignAccount,
+					require_weight_at_most: Weight::from_parts(1_000_000_000, 1024 * 1024),
+					call: remark.encode().into(),
+				}
+			]),
+		));
+	});
+
+	// Execute remote transact and verify that `Remarked` event is emitted.
+	ParaA::execute_with(|| {
+		use parachain::{RuntimeEvent, System};
+		assert!(System::events().iter().any(|r| matches!(
+			r.event,
+			RuntimeEvent::System(frame_system::Event::Remarked { .. })
+		)));
+	});
+}
+
+#[test]
+fn para_to_para_native_transfer_and_back() {
```

Comment:
```suggestion
fn laos_para_to_astar_para_reserve_transfer_and_back() {
```
give more context as we specify parachains involved and also the type of tranfer we are going to do

Reply:
I would personally refrain from using `astar` name anywhere in `simulator` tests. Because, we are not using anything from `astar`, it's simply two mock runtimes. 

I think it would make sense more if we do that in `chopsticks` or `emulator` tests